examples/src/main/java/org/apache/spark/examples/ml/JavaCountVectorizerExample.java

```diff
@@ -54,6 +54,7 @@ public static void main(String[] args) {
       .setOutputCol("feature")
       .setVocabSize(3)
       .setMinDF(2)
+      .setBinary(true)
```

Comment:
@MLnick Since we introduce Binary toggle in the doc, I thought it would make sense to show how to set it. Do you want me to remove it from all examples?